src/presentation/pages/MainPage.js

```diff
@@ -1,41 +1,62 @@
 import '../styles/styleSheets/css//MainPage.css';
 // eslint-disable-next-line
-import {BrowserRouter as Router, Link, Route, Routes} from 'react-router-dom';
+import {BrowserRouter as Router, Link, Route, Routes, useLocation, useNavigate} from 'react-router-dom';
 import React, {useState} from "react";
-import TabGroup, { activeTab} from "../styles/styledComponents/StyledHeader.js";
+import TabGroup from "../styles/styledComponents/StyledHeader.js";
 import * as animationConstants from "../styles/framerMotionComponents/AnimationConstants.js";
 import {StyledRect} from "../styles/styledComponents/StyledRect.js";
 import { motion } from 'framer-motion';
 import { SidebarDots } from '../styles/framerMotionComponents/AnimatedComponents';
 import { HoverButtonClick } from '../styles/framerMotionComponents/HoverButton';
+import Modal from '../styles/framerMotionComponents/Modal.js'
 
 /* define main page function 
 Page responsible for displaying the 3 prompt choices and allowing user to flip through and select one
 Also provides option to view help page and reupload info
 
 */
+
+//This is the type of the voiceflow blocks
+export const tabNames = ["Option A", "Option B"];
```

Comment:
Are we still using Option A and Option B for tab names? I thought we were using General and Specific.